unison-src/transcripts/idempotent/fix-ls.md

```diff
@@ -14,8 +14,7 @@ foo.bar.subtract x y = x Int.- y
   Loading changes detected in scratch.u.
 
   I found and typechecked these definitions in scratch.u. If you
-  do an `add` or `update`, here's how your codebase would
-  change:
+  do an `update`, here's how your codebase would change:
 
     ⍟ These new definitions are ok to `add`:
```

Comment:
Should probably update `add` -> `update` in this message too

Reply:
Oops! Yeah